examples/SignalingTestApp/SignalingTestApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,344 @@
+// !$*UTF8*$!
+{
```

Comment:
Isn't SignalingTestApp the trouter stuff? Should this be merged?

Reply:
Ah yes... good catch. This is just the base app Gloria created for testing but I will remove it since we are not including signalling changes yet.

Reply:
I think this does highlight some issues with our "one feature branch" approach, but I agree. For now we should remove the test app.